doc/integrator/create_application.rst

```diff
@@ -84,12 +85,12 @@ it later.
             --env=EXTENT="2420000,1030000,2900000,1350000" \
             camptocamp/geomapfish-tools:${GEOMAPFISH_VERSION} \
             run $(id -u) $(id -g) /src \
-            pcreate -s create \
-            --package-name ${GEOMAPFISH_PACKAGE} ${GEOMAPFISH_PROJECT}
+            pcreate --scaffold=create \
+            --package-name=${GEOMAPFISH_PACKAGE} \
+            ${GEOMAPFISH_PROJECT}
 
 This will create a directory named ``<project>`` that will be next to the
-``c2cgeoportal`` directory, or to the directory of the application you are
-creating this application from.
+directory of the application you are creating this application from.
```

Comment:
This will create a directory named ``<project>`` in you current directory.

Reply:
+1 :-)